### ISSUE TITLE
关于mask_batch的问题

### ISSUE THREAD

**Issue by TangShengqin** [twangnh/Distilling-Object-Detectors#10]
repo主你好，我有下面3个问题，希望能得到你的解答：
1. anchor_target_layer.py中，mask_per_gt = torch.sum(IOU_map[i][:,:,:,k]>max_iou_per_gt, dim = 2) 这里将HWK在K维度上求和，这个操作和论文中提到的 ”With F, we filter the IOU map to keep those larger then F locations and combine them with OR operation to get a W × H mask“ 不匹配呀？
2. 另外，论文中的公式3，imitation loss weight balancing factor这个值取值为多少呢？代码中默认值是0.01
3. repo主有没有试过当K等于1，HW上一个GT框可能就只有一个Loc能与之匹配，这种情况下做mimic还有效果吗？
